File: creational/python/builder/HighPackCarBuilder.py
from CarBuilder import CarBuilder

class HighPackCarbuilder(CarBuilder):
    def __init__(self):
        CarBuilder.__init__(self)

    def buildEngine(self) ->None:
        self._car.setEngine("w12")

    def buildInterior(self) ->None:
        self._car.setInterior("full pack interior")

    def buildHeadLight(self) ->None:
        self._car.setHeadlight("matrix")

    def buildSunroof(self) -> None:
        self._car.setSunroof("panoramical glass ceiling")

File: creational/java/builder/src/HighPackCarBuilder.java
public class HighPackCarBuilder  extends CarBuilder{
    @Override
    public void buildEngine() {
        this.car.setEngine("w12");
    }

    @Override
    public void buildHeadLight() {
        this.car.setHeadlight("matrix");
    }

    @Override
    public void buildInterior() {
        this.car.setInterior("full pack");
    }

    @Override
    public void buildSunroof() {
        this.car.setSunroof("panoramic glass ceiling");
    }
}
